scripts/download_backtest_data.py

```diff
@@ -27,15 +30,36 @@
 
 print(f'About to download pairs: {PAIRS} to {dl_path}')
 
-# Init Bittrex exchange
+# Init exchange
 exchange._API = exchange.init_ccxt({'key': '',
                                     'secret': '',
-                                    'name': 'bittrex'})
+                                    'name': args.exchange})
 
 for pair in PAIRS:
     for tick_interval in TICKER_INTERVALS:
         print(f'downloading pair {pair}, interval {tick_interval}')
-        data = exchange.get_ticker_history(pair, tick_interval)
+
+        since_time = None
+        if args.days:
+            since_time = int((time.time() - args.days * DAY_SECCONDS) * 1000)
+
+        # download data until it reaches today now time
+        data = []
+        while not since_time or since_time < (time.time() - 10 * MIN_SECCONDS) * 1000:
```

Comment:
I think this loop should be moved into `get_ticker_history` - so it can be used by the backtesting-script as well (when using --refresh-pairs-cached).